packages/chrome/src/views/Chrome.spec.js

```diff
@@ -6,21 +6,20 @@
  */
 
 import React from 'react';
-import { shallow } from 'enzyme';
-import { shallowWithTheme } from '@zendeskgarden/react-testing';
+import { render, renderRtl } from 'garden-test-utils';
```

Comment:
Only issue with this is it create a sense that this is an external package and might be a bit of misdirection as to where it comes from.

Reply:
I agree that this may become confusing, but I think this abstraction isn't premature.  From their [setup docs](https://testing-library.com/docs/react-testing-library/setup#configuring-jest-with-test-utils) this seems to be a common practice and we already do this with our enzyme test utilities.

With the file being the same name as the import I hope it would be discoverable as well.